examples/experimental/wsl.yaml

```diff
@@ -0,0 +1,27 @@
+# ===================================================================== #
+# BASIC CONFIGURATION
+# ===================================================================== #
+
+# Default values in this YAML file are specified by `null` instead of Lima's "builtin default" values,
+# so they can be overridden by the $LIMA_HOME/_config/default.yaml mechanism documented at the end of this file.
+
+# VM type: "qemu" or "vz" (on macOS 13 and later).
+# The vmType can be specified only on creating the instance.
+# The vmType of existing instances cannot be changed.
+# 🟢 Builtin default: "qemu"
+vmType: wsl2
+
+# OpenStack-compatible disk image.
+# 🟢 Builtin default: null (must be specified)
+# 🔵 This file: Ubuntu 23.04 Lunar Lobster images
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "file://C:/Users/Administrator/Downloads/fedora-38-cloud-rootfs.tar"
+  arch: "x86_64"
+  digest: "sha256:112400224d801c3781c1e301408685920ba980024bfb20877628a0bd82fdc574"
+
+mountType: wsl
+
+containerd:
+  system: true
+  user: false
```

Comment:
Can be removed, as it corresponds to the default values

Reply:
I think default is `user: true`, not `system: true`?